ntc_templates/templates/edgecore_show_interfaces_brief.textfsm

```diff
@@ -0,0 +1,19 @@
+Value INTERFACE (\S+\s+\d+\s*/\s*\d+)
+Value NAME (.*?)
+Value STATUS (\S+)
+Value PVID (\d+)
+Value PRI (\d+)
```

Comment:
What is `PRI` in this instance? Is it priority or something related to a PRI on the T1 world?

Reply:
This is "Priority for Untagged Traffic". Changed the name PRI -> PRIORITY